src/mom5/ocean_param/vertical/ocean_vert_mix.F90

```diff
@@ -2784,8 +2787,10 @@ end subroutine watermass_diag_init
 !
 subroutine vert_mix_coeff(Time, Thickness, Velocity, T_prog,   &
                           T_diag, Dens, swflx, sw_frac_zt, pme,&
-                          river, visc_cbu, visc_cbt, diff_cbt, hblt_depth, do_wave)
-
+!dhb599:                          river, visc_cbu, visc_cbt, diff_cbt, hblt_depth, do_wave)
+                          river, visc_cbu, visc_cbt, diff_cbt, hblt_depth, Ice_ocean_boundary, do_wave)
+  type(ice_ocean_boundary_type),  intent(in)    :: Ice_ocean_boundary
+!
```

Comment:
This change makes me a little wary.  `vert_mix_coeff` is used in all sorts of models, but it seems they have changed the arguments here.

Reply:
It looks like the extra argument was added here solely to pass to `vert_mix_kpp_mom4p1`.  At the least, this probably needs to be optional.